src/views/AccountingOverview/AccountingOverviewV2.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { ReactNode, useState } from 'react'
```

Comment:
Not sure if this file is needed at all. The difference between "v1" and "v2" are not large but noticeable.
To discuss: how much we can modify layouts (views) provided with the package in terms of client awareness for such changes?